### ISSUE TITLE
Doesn't work

### ISSUE THREAD

**Issue by Tripazzy** [jonmoody/nes-tutorial#1]
I do what you tell me to do [here](http://thevirtualmountain.com/nes/2017/03/13/graphics-and-creating-backgrounds.html), but the screen stays black. I even copy-and-pasted it straight from here and it didn't work. It's in tutorial 2.

**Comment by jonmoody** [comment]
Did you checkout the tutorial-2 branch to see if that works? If that works, it is possible I missed something in my explanation. If checking out that branch doesn’t work, could you tell me which emulator you’re using? Either way, I am away from computers for the weekend but tomorrow I can try to figure out what’s going on.

**Comment by Tripazzy** [comment]
Tutorial-2 was the one I copy-and-pasted from when it didn't work.

**Comment by jonmoody** [comment]
The tutorials are meant to be done sequentially and build upon the previous one. Did you follow and complete tutorial-1? That contains a lot of setup code that we need for tutorial-2.

**Comment by Tripazzy** [comment]
Oh, I was doing it as a separate project. I'll try that.

**Comment by jonmoody** [comment]
Yeah, definitely let me know if that helps. This page will show them all in order: [http://thevirtualmountain.com/nes/](http://thevirtualmountain.com/nes/)

If that works, I'll be sure to update the tutorials to be more clear and call that out.
